docs/user/translating.rst

```diff
@@ -299,9 +301,10 @@ translated string is not too long:
 Glossary
 --------
 
-Each project can have an assigned glossary for any language as a shorthand for storing terminology.
-Consistency is more easily maintained this way.
-Terms from the currently translated string can be displayed in the bottom tabs.
+Each project can have an assigned glossary for any language as a shorthand for storing
+terminology. Consistency is more easily maintained this way.
+Terms from the glossary containing words from the currently translated string can be
```

Comment:
The bottom tabs? In the context menu?

Reply:
It is probably just heavily outdated. The glossary is shown in the sidebar.